FND/Places/Addresses.rdf

```diff
@@ -763,6 +763,12 @@
 		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/common-data-file-format/lei-cdf-format/lei-cdf-format-version-2-1</fibo-fnd-utl-av:adaptedFrom>
 	</owl:ObjectProperty>
 	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-plc-adr;hasLocaleDesignation">
+		<rdfs:subPropertyOf rdf:resource="&lcc-lr;hasName"/>
+		<rdfs:label>has locale designation</rdfs:label>
+		<skos:definition>location-specific appelation (name) for something</skos:definition>
+	</owl:DatatypeProperty>
```

Comment:
If it's meant to be a location-specific name, where do you specify what the location is that it's specific to? This seems to be missing something significant!